docs/content/direct/wec-registration.md

```diff
@@ -1,5 +1,307 @@
 # Registering a Workload Execution Cluster
 
-This document will tell users what how to register a WEC in an ITS.
+This document explains how to register a Workload Execution Cluster (WEC) with an Inventory and Transport Space (ITS) in KubeStellar.
 
-TODO: write this.
+## Overview
+
+Registering a WEC in an ITS is the same process as registering a managed cluster with an OCM hub cluster. KubeStellar uses Open Cluster Management (OCM) for cluster registration and management.
+
+
+### Terminology Mapping
+
+| OCM Term             | KubeStellar Equivalent                        |
+|----------------------|-----------------------------------------------|
+| **OCM Hub**          | **KubeStellar ITS** (Inventory and Transport Space) |
+| **OCM Managed Cluster** | **KubeStellar WEC** (Workload Execution Cluster) |
+| **OCM Agent**        | **OCM Agent** (same component, installed on WEC) |
+
+For the complete OCM registration process, refer to the [official Open Cluster Management documentation](https://open-cluster-management.io/docs/getting-started/installation/register-a-cluster/).
+
+## Prerequisites
+
+Before registering a WEC, ensure you have:
+
+1. **A running Kubernetes cluster** that will serve as the WEC
+2. **Network connectivity** between the WEC and the ITS
+3. **`kubectl` access** to both the WEC and ITS
+4. **`clusteradm` CLI tool** installed ([installation guide](https://open-cluster-management.io/docs/getting-started/installation/start-the-control-plane/))
```

Comment:
There are several containers and (virtual or real) machines in sight, so it would be helpful to say _where_ this tool needs to be installed.